history.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: History
+slug: history
+
+---
+
+## One Laptop Per Child
+
+One Laptop Per Child ([OLPC](https://laptop.org/)) was proposed by MIT in 2005 to [fight the gap](https://laptop.org/aboutolpc/) in access to technology in education for underprivileged children.
+
+Stephen Jacobs, director of RIT's Lab for Technological Literacy (LTL) at the time, was interested in the program and acquired three OLPC XO 1.0 laptops in 2007 via the first "Give One, Get One" program. 
+In January 2009, Rochester Institute of Technology had an honors class that developed educational games for the OLPC program. That was the earliest documented birth of Open-Source @ RIT. 
+Open-source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class. 
```

Comment:
```suggestion
Open Source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class.
```

Open Source in capitals because it is referring to a proper noun in this case: https://opensource.org/definition-annotated